lib/view_models/evaluate_form/evaluationFormButton_view_model.dart

```diff
@@ -131,4 +131,11 @@ class EvaluationFormViewModel {
     }
     return false;
   }
+
+  bool disableRecoveryReadinessButton(bool check) {
+    if (check == false) {
+      return true;
+    }
+    return false;
+  }
```

Comment:
change to `bool disableRecoveryReadinessButton(bool check) => return !check;`

Reply:
Solved in d11894d ka